packages/nuxt/src/pages/utils.ts

```diff
@@ -573,6 +573,10 @@ async function createClientPage(loader) {
         metaRoute.children = route.children
       }
 
+      if (route.props) {
+        metaRoute.props = route.props
+      }
```

Comment:
nice catch.

I think instead we need to set props value on the generated route meta.

does this work for you?

Reply:
Sorry for late. I will try this next day..

Reply:
Do you think this is right ? Add `props: serializeRouteValue(page.props)`

Reply:
perfect!